shopify/product-reviews-template/web/routes/index.jsx

```diff
@@ -0,0 +1,101 @@
+import { AutoTable } from "@gadgetinc/react/auto/polaris";
+import {
+  BlockStack,
+  Box,
+  Card,
+  Layout,
+  Page,
+  Text,
+  Tooltip,
+} from "@shopify/polaris";
+import { api } from "../api";
+import { ApprovalButton, Stars } from "../components";
+import { useAppBridge } from "@shopify/app-bridge-react";
+import { useState } from "react";
+
+export default function () {
+  const { toast, modal } = useAppBridge();
+  const [modalContent, setModelContent] = useState("");
+
+  return (
+    <Page>
```

Comment:
Can we include a Page title? Looks a tad barren right now
![CleanShot 2024-10-29 at 09 05 43@2x](https://github.com/user-attachments/assets/32a726c7-402f-4f72-91fe-b8b9d3c6ead1)